templates/text-gen-service/service.yaml

```diff
@@ -0,0 +1,11 @@
+name: anyscale-text-generation-template
+ray_serve_config:
+  import_path: server:deployment
+  runtime_env:
+    # This working dir is relative to the working dir when we run this file
+    working_dir: "."
+    excludes: 
+    - "deploy"
+    - "aviary/frontend"
+    - "**.DS_Store"
+    - ".git"
```

Comment:
Are these all needed? I'm not very familiar with this part so I may be missing some context but these feel a bit "random" to need to specify.

Reply:
this can carry foward from another project of mine. cleaned up now